package.json

```diff
@@ -53,6 +53,7 @@
     "@types/sinon-chai": "^3.2.5",
     "chai": "^4.2.0",
     "ethereum-waffle": "^3.0.0",
+    "ethers": "^5.0.0",
```

Comment:
What is this change used for?

Reply:
Force of habit to mirror peerDeps in devDeps for local development, but feel free to remove, I'm not sure there's a good reason in this case.

Reply:
I just noticed ethers is both a dependency and a peer dependency. Is that intentional?

Reply:
It needs to be a dev dependency and a peer dep. Shouldn't be a primary dep. That might be a bug.

Reply:
Removed for this PR. I'll take a look at why we made it part of the primary deps.